ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -283,6 +283,7 @@ func convertLogging(logConfig *ecs.LogConfiguration) *composeV3.LoggingConfig {
 	if logConfig == nil {
 		return nil
 	}
+	log.Warning("AWS logging will not be supported")
```

Comment:
I would maybe interpolate this to use the `LogDriverAwslogs` constant in the aws-go-sdk.

Reply:
done!